components/atom/label/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@s-ui/react-atom-label",
```

Comment:
i was wondering if package name should be `@s-ui/sui-atom-label`

looks like most of the packages use `sui` prefix, even if there are some with `react`:
https://github.com/SUI-Components/sui-components/blob/master/components/atom/spinner/package.json#L2
https://github.com/SUI-Components/sui-components/blob/master/components/atom/badge/package.json#L2

Reply:
Enablers decided to rename the components to @s-ui/react-[category]-[name] but it isn't mandatory until we do the proper changes in the studio to automatically set the right name